internal/pkg/model/path.go

```diff
@@ -22,8 +22,8 @@ func NewAbsPath(parentPath, objectPath string) AbsPath {
 	return AbsPath{parentPath: Path(parentPath), parentPathSet: true, RelativePath: Path(objectPath)}
 }
 
-func (p AbsPath) DeepCopy(_ deepcopy.TranslateFunc, _ deepcopy.Steps, _ deepcopy.VisitedMap) AbsPath {
-	return p
+func (p AbsPath) DeepCopy(_ deepcopy.TranslateFunc, _ deepcopy.Steps, _ deepcopy.VisitedPtrMap) (AbsPath, deepcopy.CloneNestedFn) {
+	return p, nil
```

Comment:
OpenApi schema moze mat referencie na schemy, medzi sebou ... v Go je to potom tak, ze sa jeden pointer vyskytuje viac krat, ked prechadzame schemu do hlbky.

To sme v `deepcopy` nepodporovali, tak som pridal podporu.